tee-worker/omni-executor/native-task-handler/src/lib.rs

```diff
@@ -643,17 +643,53 @@ async fn handle_native_task<
 		},
 		NativeTask::PumpxExportWallet(
 			sender,
-			_maybe_google_code,
+			maybe_google_code,
 			pumpx_wallet_chain,
 			pumpx_wallet_index,
 			expected_wallet_address,
 		) => {
+			if let Some(ref google_code) = maybe_google_code {
+				let storage = PumpxJwtStorage::new(ctx.storage_db.clone());
+				let Some(access_token) =
+					storage.get(&(sender.to_omni_account(), AUTH_TOKEN_ACCESS_TYPE))
+				else {
+					let response = NativeTaskResponse::Err(NativeTaskError::InternalError);
+					if response_sender.send(response.encode()).is_err() {
```

Comment:
Can we log/identify error reason somehow so later if this happens we know why?

Reply:
It doesn't need to be propagated to client - it can be logged.
Just in case we experience this we can know the reason why request has failed - just by looking into logs for example.

Reply:
Sure I'll add them